src/main/java/thePackmaster/actions/bitingcoldpack/ActivateGlaciateAction.java

```diff
@@ -0,0 +1,30 @@
+package thePackmaster.actions.bitingcoldpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.watcher.VigorPower;
+
+import static thePackmaster.util.Wiz.*;
+
+public class ActivateGlaciateAction extends AbstractGameAction {
```

Comment:
Why is this an action? Just because you wanted to be absolutely sure the power icon flashes at the correct time? 
You can put that into an anonymous action.